frontend/app/src/composables/balances/index.ts

```diff
@@ -7,7 +7,7 @@ import { One } from '@/utils/bignumbers';
 import { uniqueStrings } from '@/utils/data';
 import { type AllBalancePayload } from '@/types/blockchain/accounts';
 
-export const useBalances = () => {
+export const useBalances = createSharedComposable(() => {
```

Comment:
ah damn, so basically the debounce didn't work because each use was a different watcher